opal/mca/accelerator/cuda/accelerator_cuda_component.c

```diff
@@ -34,7 +34,7 @@ static int accelerator_cuda_open(void);
 static int accelerator_cuda_close(void);
 static int accelerator_cuda_component_register(void);
 static opal_accelerator_base_module_t* accelerator_cuda_init(int *priority);
-static void accelerator_cuda_finalize(void);
```

Comment:
Overall commit feedback, null component should be a separate commit from changing selection logic. Would make it much easier for me to append to my commits.

Reply:
done